src/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10.000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def age_of?
+    @age >= 18
+  end
+
+  def can_use_services?
+    age_of or @parent_permission
+  end
```

Comment:
- [ ] Please this is a public method that returns true if person is of age or if they have permission from parents. Also fix the `is_of_age?()` and the `or` operator like so `is_of_age?() || @parent_permission`. Please kindly fix that.

Reply:
By default, can_use_services is a public method in ruby?

Reply:
Hello @Hector096, you placed it after the `private` keyword, that's what made it private. Thanks for correcting it.